io.catenax.message_header/1.0.0/MessageHeaderAspect.ttl

```diff
@@ -0,0 +1,103 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
```

Comment:
according to convention change namespace to 

io.catenax.shared.message_header/1.0.0/MessageHeaderAspect.ttl

(models, which are only used in conjunction with other models are moved to the shared namespace)

Reply:
namespace has been moved